packages/api/src/resolvers/article/index.ts

```diff
@@ -822,8 +822,12 @@ export const bulkActionResolver = authorized<
         },
       })
 
-      // the query size is limited to 255 characters
-      if (!query || query.length > 255) {
+      // the query size is limited to 4000 characters to allow for 100 items
+      if (!query || query.length > 4000) {
```

Comment:
@jacksonh Changed to max 4000 characters here

Reply:
🙏